modules/tinymce/src/themes/silver/main/ts/ui/menus/menu/MenuMovement.ts

```diff
@@ -20,7 +20,13 @@ export const deriveMenuMovement = (columns: number | 'auto', presets: Toolbar.Pr
       }
     } as MenuTypes.MenuGridMovementSpec;
   } else {
-    const rowClass = presets === 'color' ? 'tox-swatches__row' : 'tox-collection__group';
+    const rowClass: Record<Toolbar.PresetTypes, string>[Toolbar.PresetTypes] = {
```

Comment:
This will fail with a type error if we add new ones to PresetTypes right?

Reply:
yes, if you try to remove one of the rows in that JSON you get an error

Reply:
That's not what Spocke asked. If a new string is added to `Toolbar.PresetTypes` will this code throw an error?

Maybe a `switch` statement without a `default` branch would be better.

Reply:
sorry I didn't explain it well, I did the experiment of removing a raw because it's equivalent of having a new string in `Toolbar.PresetTypes` and I thought it was faster too, but actually adding 1 string to `PresetTypes` is fast too :sweat_smile: (I made a misunderstanding for nothing sorry)

Reply:
I confirmed via slack that we're on the same page